fs_mark/fs_mark.py

```diff
@@ -0,0 +1,159 @@
+    END = r"Average Files/sec:.+?\r+\n"
+    ERR = r".+?(?P<err>EIO|ENOSPC|ENOMEM).+?"
+    NO_CONT_BLOCK = r"(?P<msg>Lack of contiguous memory block of size \d+ bytes.+?)\r+\n"
+
+    while True:
+        if loop_start and loop_end:
+            loop_time = 3 * (loop_end - loop_start)
+            loop_start = None
+            loop_end = None
+
+        try:
+            idx = dut.expect([
+                NAME,
+                MSG_LINE,
+                NO_SPC,
+                END,
+                NO_CONT_BLOCK,
+                ERR
+            ], timeout=loop_time)
+            parsed = dut.match.groupdict()
+
+        except pexpect.TIMEOUT:
+            test_msg = 'Got timeout, probably fs hang-up'
+            test_status = Status.FAIL
+            break
+
+        if idx == 0:
+            test_name = parsed["name"]
+            loop_start = time.time()
+        elif idx == 1:
+            first_loop_done = True
+            splitted_line = parsed["line"].split()
+            f_use = splitted_line[0]
+            count = splitted_line[1]
+            size = splitted_line[2]
+            files_sec = splitted_line[3]
+            app_overhead = splitted_line[4]
+
+            line_dict = {}
+            line_dict['creatMin'] = splitted_line[5]
+            line_dict['creatAvg'] = splitted_line[6]
+            line_dict['creatMax'] = splitted_line[7]
+            line_dict['writeMin'] = splitted_line[8]
+            line_dict['writeAvg'] = splitted_line[9]
+            line_dict['writeMax'] = splitted_line[10]
+            line_dict['closeMin'] = splitted_line[17]
+            line_dict['closeAvg'] = splitted_line[18]
+            line_dict['closeMax'] = splitted_line[19]
+
+            # If files are kept, fs_mark will unlink them so unlink time can be caught
+            if not ('-k' in test_name or '-L' in test_name or '-F' in test_name):
+                line_dict['unlinkMin'] = splitted_line[20]
+                line_dict['unlinkAvg'] = splitted_line[21]
+                line_dict['unlinkMax'] = splitted_line[22]
+
+            for name, value in line_dict.items():
+                if not t_data.timing_dict[(target, name)][0] <= int(value) <= t_data.timing_dict[(target, name)][1]:
+                    test_status = Status.FAIL
+                    test_msg += ''.join(('\n\t', name, ' exec time - ', value, ' out of range [',
+                                         str(t_data.timing_dict[(
+                                             target, name)][0]), ' - ',
+                                         str(t_data.timing_dict[(target, name)][1]), ']'))
+
+            if test_status == Status.FAIL:
+                test_msg += "\n\n\tF_Use%: " + str(f_use)
+                test_msg += "\n\tCount: " + str(count)
+                test_msg += "\n\tSize: " + str(size)
+                test_msg += "\n\tFiles/sec: " + str(files_sec)
+                test_msg += "\n\tApp overhead: " + str(app_overhead) + "\n\t"
```

Comment:
What do you think about this? :
```suggestion
		    test_msg += '\n\t'.join((
		        f"F_Use%: {f_use}",
		        f"Count: {count}",
		        f"Size: {size}",
		        f"Files/sec: {files_sec}",
		        f"App overhead: {app_overhead}\n\t"
		    ))
```